tests/test_interpolatedimage.py

```diff
@@ -1036,6 +1036,72 @@ def test_stepk_maxk():
     t2 = time.time()
     print 'time for %s = %.2f'%(funcname(),t2-t1)
 
+def test_kround_trip():
+    a = final
+    real_a, imag_a = a.drawKImage()
+    b = galsim.InterpolatedKImage(real_a, imag_a)
+
+    # Check picklability
+    do_pickle(b)
+    do_pickle(b, lambda x: x.drawImage())
+    # do_pickle(b.SBProfile)
+    # do_pickle(b.SBProfile, lambda x: repr(x))
```

Comment:
These should work too.  Are the commented out because they failed?  Or too slow?